createAPIJar.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<project name="v9VersionAndJarBuild" default="jar" basedir=".">
```

Comment:
Fixed typo in ant script that creates the "test" jar to avoid confusion with the ant script it was copied from